### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by rahearn**
First 4 issue templates:

* User Story
* Task
* Bug Report
* Team Manager checklist

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: User Story
+about: Suggest an idea for this project
+title: ''
+labels: research, user-story
```
Comment (rahearn) [inline .github/ISSUE_TEMPLATE/user-story.md:5]: New user stories are automatically given the `research` label, as I assume stories should first at least trigger us to ask "is research needed on this?"

Happy to remove that if we'd rather proactively add that label to stories that are going through research instead.

Comment (lauranash18f) [reply .github/ISSUE_TEMPLATE/user-story.md:5]: Could we do "research?" — show that it's not definitive?

**Review by lauranash18f — APPROVED** [review]
We may need more sectioning to user stories once we start using them, but this is a great starting point (and they are distinct from one another, imo.)

**Review by hartsick — APPROVED** [review]
looks good to me!

seeing the user story template makes me feel like it would be helpful to have a conversation about development process once we identify who will be working on which tracks.

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 1 above)
Comment (rahearn) [reply .github/ISSUE_TEMPLATE/user-story.md]: For context, this conversation continued into slack: https://gsa-tts.slack.com/archives/C02K6V0UNE9/p1637600423002900

**Pull request merged**